src/platforms/common/configuration/options.mdx

```diff
@@ -96,6 +96,12 @@ This variable controls the total amount of breadcrumbs that should be captured.
 
 </ConfigKey>
 
+<ConfigKey name="max-cache-items" supported={[ "dotnet", "android"]} >
```

Comment:
it's called `cacheDirSize` on Android though

Reply:
remove `android` for now please, till we get https://github.com/getsentry/sentry-java/issues/1162 done

Reply:
Ah didn't know, that this is not done yet. Thanks